swan-lake/resources/featured-scenarios/write-a-restful-api-with-ballerina.md

```diff
@@ -275,95 +304,77 @@ public type CovidEntry record {|
     decimal active;
 |};
 
-public final table<CovidEntry> key(iso_code) covidTable = table [
+final table<CovidEntry> key(iso_code) covidTable = table [
     {iso_code: "AFG", country: "Afghanistan", cases: 159303, deaths: 7386, recovered: 146084, active: 5833},
     {iso_code: "SL", country: "Sri Lanka", cases: 598536, deaths: 15243, recovered: 568637, active: 14656},
     {iso_code: "US", country: "USA", cases: 69808350, deaths: 880976, recovered: 43892277, active: 25035097}
 ];
 
-public type ConflictingIsoCodesError record {|
+type IsoCodeConflict record {|
     *http:Conflict;
-    ErrorMsg body;
+    string body;
 |};
 
-public type InvalidIsoCodeError record {|
+type IsoCodeNotFound record {|
     *http:NotFound;
-    ErrorMsg body;
-|};
-
-public type ErrorMsg record {|
-    string errmsg;
+    string body;
 |};
 ```
 
-- It is always a good practice to document your interfaces. However, this example has omitted documentation for brevity. Nevertheless, any production-ready API interface must include API documentation. 
-- You can also try generating an OpenAPI specification for the written service by executing the following command, which creates a `yaml` file in the current folder.
-
 ## Run the service
 
-In the terminal, navigate to the `covid19` directory, and execute the command below to run the service package.
+Use the [**Run**](/learn/vs-code-extension/run-a-program/) CodeLens of the VS Code extension to build and run the service, as shown below.
 
-> **Info**: The console should have warning logs related to the isolatedness of resources. It is a built-in service concurrency safety feature of Ballerina.
+![Run the service](/learn/images/featured-scenarios/write-a-restful-api-with-ballerina/run-the-service.gif)
 
+>**Info:** Alternatively, you can run this service by navigating to the project root (i.e., the `restful-service` directory) and executing the `bal run` command. The console should have warning logs related to the isolatedness of resources. It is a built-in [service concurrency safety](https://ballerina.io/learn/by-example/#concurrency-safety) feature of Ballerina.
 
-```
-$ bal run
-```
-
-You view the output below.
+You should see the output similar to the following.
```

Comment:
Same comment re: the.

Reply:
This should be grammatically correct to have the `an` article as follows.

`You should see an output similar to the following.`

Reply:
Updated.